providers/base/units/firmware/jobs.pxu

```diff
@@ -105,3 +105,15 @@ plugin: attachment
 depends: firmware/fwts_dump
 command:
   [ -f "$PLAINBOX_SESSION_SHARE/acpidump.log" ] && gzip -c "$PLAINBOX_SESSION_SHARE/acpidump.log"
+
+id: firmware/fwupdmgr_get_devices
+plugin: attachment
+category_id: com.canonical.plainbox::firmware
+_summary: Collect the device firmware update information
+requires:
+    executable.name == "fwupdmgr"
+command:
+    snap_var="$SNAP"
+    unset SNAP
+    fwupdmgr get-devices --json
+    export SNAP="$snap_var"
```

Comment:
The saving and restoring of the $SNAP is not needed, as any changes to the environment is local to the  command being run.

Reply:
Thanks for the comments, I have removed the step to saving and restoring the SANP env var.